Fonts/TomThumb.h

```diff
@@ -43,429 +43,435 @@
 ** https://gist.github.com/skelliam/322d421f028545f16f6d
 ** William Skellenger (williamj@skellenger.net)
 ** Twitter: @skelliam
-**
+** 
+** Jan 09, 2020: Bitmaps now compressed, to fix the bounding box problem,
```

Comment:
Can you attach the code you used to generate the font header?
It's good to be able to generate the header from the upstream font source BDF file.
We previously had a link to the @skelliam bdf2adafruit.py gist.

I would like to suggest that the @skelliam bdf2adafruit.py be modified or rewritten to produce your improved header format, and then checked into the fontconvert/ directory here, so we never lose access to it.

Reply:
I changed the bdf2adafruit.py gist a little bit, to generate the compressed set. ;-)
Will commit it to /fontconvert folder.